csharp/src/Bargreen.Tests/InventoryControllerTests.cs

```diff
@@ -10,11 +11,13 @@ namespace Bargreen.Tests
 {
     public class InventoryControllerTests
     {
+        private InventoryService InventoryService;
+
         [Fact]
-        public void InventoryController_Can_Return_Inventory_Balances()
+        public async void InventoryController_Can_Return_Inventory_Balances()

```

Comment:
async methods should return a Task. 
(There are exceptions to this rule, but it's a complicated use case)